code/sorting/quick_sort/quicksort.rs

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+ 
+// This algorithm taken from https://github.com/servo/rust-quicksort
+
+//! In-place sorting.
```

Comment:
Kindly remove the above comments and add this comment  
/* Part of Cosmos by OpenGenus Foundation */
with every code you commit !

Reply:
The above comments do not need to be removed, but the header ones need to be added.

Reply:
Shall i add Cosmos part at the begining of file ?

Reply:
It doesn't matter where it is, as long as it is near the top @canberkkoc.